.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push]
+
+jobs:
+  rspec:
+    name: Run rspec
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-11, windows-2022]
+        ruby: [2.6, 2.7, 3.0, 3.1]
+    runs-on: ${{ matrix.os }}
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.99.0
```

Comment:
なるほど最新のタグのバージョンなんですね https://github.com/ruby/setup-ruby/tags

`v1` とだけしておけば自動的にバグフィックスが適用される的なことが書いてあるけど、どっちがいいんですかね 🤔  反対にバージョン指定だと自動的にはならないっぽい

> It is highly recommended to use ruby/setup-ruby@v1 for the version of this action. This will provide the best experience by automatically getting bug fixes, new Ruby versions and new features.

https://github.com/ruby/setup-ruby#versioning

Reply:
おお、GitHub 公式では sha 指定推奨なんですね、ありがとうございます！